tests/strategies/dictionaries/test_trie_dictionary_factory.py

```diff
@@ -0,0 +1,234 @@
+from collections.abc import ItemsView, KeysView
```

Comment:
Could we somehow reuse generic `DictionaryFactory` tests for all the implementations?

Reply:
For common functionality that would be helpful and I considered implementing dictionary agnostic tests, but concluded that this would be out-of-scope for this PR.